src/main/java/sirius/kernel/xml/SOAPClient.java

```diff
@@ -169,6 +170,18 @@ public SOAPClient withActionPrefix(@Nonnull String prefix) {
         return this;
     }
 
+    /**
+     * Defines a custom value for the "Content-Type" HTTP header to use for requests. By default, the header is set
+     * to "text/xml".
```

Comment:
**Optional:** Why not define `text/xml` as a constant like `DEFAULT_CONTENT_TYPE` and link it here? Then the info is less likely to become outdated when the default is somehow changed.